src/config/hintr_version

```diff
@@ -1 +1 @@
-master
+mrc-1744
```

Comment:
This HINT branch needs to be merged first, then this hintr branch can be merged, then this can be pointed back to master.